streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/TaskMetrics.java

```diff
@@ -213,7 +272,7 @@ public static Sensor recordLatenessSensor(final String threadId,
     public static Sensor droppedRecordsSensor(final String threadId,
                                               final String taskId,
                                               final StreamsMetricsImpl streamsMetrics) {
-        return invocationRateAndCountSensor(
+        return invocationRateAndTotalSensor(
```

Comment:
This is a piggy-backed metric fix: we should use cumulativeSum than cumulativeCount for dropped records, even though today with most callees as `sensor.record()` it is effectively the same as it only increment by 1, it is still vulnerable in case we record a non-one value in the future.